src/bundle/DependencyInjection/Compiler/ComponentPass.php

```diff
@@ -13,13 +13,18 @@
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\DependencyInjection\Exception;
 
 class ComponentPass implements CompilerPassInterface
 {
-    const TAG_NAME = 'ezplatform.admin_ui.component';
+    public const TAG_NAME = 'ezplatform.admin_ui.component';
```

Comment:
should this be unified across all new PRs? it is not in PSR afair?

Reply:
I wouldn't go with that change in this PR. I see PHPStorm is proposing this change but this will introduce too many inconsistencies in the codebase.